elixir/apps/domain/test/support/mocks/www_firezone.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Domain.Mocks.WWWFirezone do
```

Comment:
WWW feels a bit weird in the module name:
```suggestion
defmodule Domain.Mocks.FirezoneWebsite do
```

Reply:
👍 updated